### DISCUSSION TITLE
가게 전화번호 & 사장님 전화번호는 어디서 가져와야 하는가

### DISCUSSION THREAD

**Discussion by gayeong0916** [team-final/albaform#61] [General]
![화면 캡처 2024-09-11 215244](https://github.com/user-attachments/assets/e83f71ea-dec8-4d81-a27f-7f00aaeffe80)

질문 1. 전화번호는 회원가입 할 때 입력할 수 있는데 지원자의 입장일 때 가게 전화번호와 사장님 전화번호는 어떻게 가져오면 될까요?
(암만 봐도 모르겠어요 정말로...)
질문 2. 공유하기 버튼(플로팅 버튼)을 눌렀을 때 해당되는 api는 없는거죠....???? 제가 못 본 거 아니죠?

**Comment by JuhyeokC** [comment]
그러게요..!

알바폼 생성할 때 `ownerId` 가 생성되는 것 같고
알바폼 상세조회할 때 응답데이터에 `ownerId` 가 있긴한데, 전화번호 관련 데이터는 없네요.

그렇다고 `ownerId` 로 데이터 요청하는 것도 없네요.

코드잇 운영진측에 문의해봐야할듯..!
